### ISSUE TITLE
Consider elevating reworked translationType and audience to scripture flavorType

### ISSUE THREAD

**Issue by mvahowe** [bible-technology/scripture-burrito#145]
Right now we have
# translationType (audience)
- First
- New
- Revision
- Study / Help Material

# translationLevel
- Basic
- Common
- Common - Literary
- Literary
- Liturgical
- Children

# projectType
## (from DBL "Paratext Resources", I think)
- Standard
- Daughter
- StudyBible
- StudyBibleAdditions
- BackTranslation
- Auxiliary
- TransliterationManual
- TransliterationWithEncoder
- Unknown

If we can agree on one thing, I hope it's dropping the "unknown" option.

Some of there options seem to me to be "not Scripture at all" and result from trying to shoehorn non-Scripture content into DBL. In SB I think we should handle these cases as distinct flavors.

I'm not sure about the difference between "first" and "new", and "Study Bible" doesn't belong in that list. (You could have a first or new project that happens to be a study Bible.)

"translationLevel" seems to be trying to do at least 2 things:
- level of language
- audience/style

eg liturgy for children is definitely a thing. I'd like to think that there's some existing way to rate language level that we could use here. My personal preference would be https://en.wikipedia.org/wiki/Common_European_Framework_of_Reference_for_Languages#Common_reference_levels which is widely used within Europe, including for sign language.

I think we could  consider an additional field for optional tags such as
- children
- dyslexic
- liturgical
- chronological

Finally, I think that, whatever these fields turn into, they should be elevated to the flavorType. They are already required for text and audio in DBL. Elevation to flavorType reduces schema duplication a little but, more importantly, allows for for more information when treating "all Scripture flavors", including x-flavors. This seems to me to be potentially of benefit for progress and coverage stats. (Some of my example tags would apply to typeset projects rather than the translation itself.)

**Comment by mvahowe** [comment]
(if "First" means something like "The First Bible Translation in This Language", that seems hugely problematic to me in SB. In the worst case we'd need an arbitration process for when two agencies disagree about whose project has seniority.)

**Comment by rdb** [comment]
I always imagined "First" should be interpreted more like "this translation is intended to meet the needs of a language group that doesn't currently have access to the scriptures", which avoids that question because it's not about whether it's factually the first translation in a language.  But this is just my own assumption.  I don't know if this is even a useful distinction.  This is another one of those historical warts where everybody forgot what the purpose of this was.

As for audience, I just had a discussion with someone who has experience with translation briefs, and I thought it interesting that he considered the following aspects of the audience of a translation:
* Region or country
* Age range or generation (Eg. children)
* Gender (there are translations for men/women specifically)
* Particular religious practice or tradition
* Literacy level and learning tradition (which includes aural learners, pre-literate,
illiterate, functionally illiterate, functionally literate, or highly literate)

That said, a translation brief is a very different thing and I don't know if these distinctions are equally useful to exist in the metadata (I imagine we want to be picking things primarily based on their usefulness for cataloguing purposes), but I felt it might be worth sharing.

**Comment by rdeblois1960** [comment]
Dan's assumption about First and New is correct. This distinction is not without problems, however, and one could ask to what extent it is relevant for SB. For certain Western language this attribute is hardly relevant at all. Even in Africa there are languages that had a Bible in the 1800s that nobody can understand anymore. At this moment only donors are interested in this question, because it helps them determine whether they want to fund the project or not.

**Comment by mvahowe** [comment]
Right, it's the "only donors" part that concerns me. If anyone starts tracking global progress in terms of where "first" projects are happening, and if funding is in any way linked to that, and given the 1800 translation issue that has been mentioned to me in other conversations, it's not hard to imagine inter-agency strife over who if anyone gets the one "first" flag. And all this gets worse with SB because we're moving to BCP47 where, eg, we could claim the first translation into English as spoken in Spain and written using a Persian script.

**Comment by klassenjm** [comment]
Minor, just for clarity -- projectType does not come from DBL "Paratext Resources", but from Paratext. When you create a project in Paratext, you specify a type.

**Comment by klassenjm** [comment]
Further on the projectType (again, just filling in, for clarity) -- I would not say that in this area people have been trying to shoehorn non-Scripture content into DBL (although some have from time to time, its true). This is simply a selection in a Paratext project configuration which then allows Paratext to provide certain types of features for the project - like a study Bible Additions project, which has very different functionality than a standard translation project.

**Comment by klassenjm** [comment]
In the spirit of trying to identify what the historical thinking was, my understanding of what was meant by the terms in translationType is below (copying Dan somewhat). I think we need to remember how far back some of these terms go. Decades (way before DBL and Registry). And the matter of competition / overlap between translation projects was not as visible (at least), if it existed at all in the same way.

- First - the first scripture translation initiated to meet the needs of a language group that doesn't currently have access to the scriptures.
- New - a new translation in a language which already has access to the scriptures - but which is most likely of a different translationLevel, or with a different audience in mind (and in this sense, helps to suggest that "first" was not to mean the 'first of its type' in a given language necessarily.
- Revision - a revision of an existing translation
- Study / Help Material - study or help material project - associated with an existing translation - not attempting to identify if it is the first or a new study material project.

I agree that study / help material is different from the others -- but it is significant in terms of the Bible Society mission of scripture engagement, and so there are projects of this type.

I see some the problems with these metadata. We certainly need a way to try to retain what is implied by our existing data while it is migrated forward into improved formats.

**Comment by mvahowe** [comment]
@klassenjm Thanks for the background information. As you will appreciate, how to migrate legacy data is something I'm quite interested in. In SB, I'd expect the "not Scripture" type of content to move to other flavorTypes. There's a diagram that might help to make this clearer at https://docs.burrito.bible/en/latest/extending.html

**Comment by rdb** [comment]
Perhaps we can redo this enum in a way that places the emphasis on the "purpose" of the translation (ie. what need it is attempting to address), rather than whether it happens to be the first or not?

I don't like this being called "type" either way, which is vague and nonspecific.
